src/features/rewards/profile/index.tsx

```diff
@@ -170,7 +170,7 @@ export default class Profile extends React.PureComponent<Props, {}> {
         {
           this.getUnverifiedInfo()
         }
-        <StyledVerifiedCheckLink onClick={onRefreshPublisher}>
+        <StyledVerifiedCheckLink onClick={onRefreshPublisher} data-test-id={'unverified-check-button'}>
```

Comment:
Are you going to need this across the other instances or just the "clickable"

Reply:
it's just so that I can trigger a click